live-examples/html-examples/content-sectioning/css/hgroup.css

```diff
@@ -0,0 +1,29 @@
+hgroup {
+    text-align: right;
+    font-family: Verdana, sans-serif;
+    padding-right: 16px;
+    border-right: 10px solid #00c8d7;
+}
+
+h1 {
+    font-size: 1.93rem;
+    margin-bottom: 0;
+}
+
+h2 {
+    font-size: 1.16rem;
+    margin-top: 0;
+}
+
+p {
+    text-align: justify;
+    font-family: Georgia, serif;
+    font-size: .95rem;
+}
+
+p::first-letter {
+    font-size: 3.2rem;
+    float: left;
+    padding-right: 8px;
+    padding-top: 4px;
+}
```

Comment:
This looks weird in Chrome - I think perhaps you need to set `line-height`? https://css-tricks.com/snippets/css/drop-caps/#article-header-id-1

Reply:
You are right, chrome needs `line-height` to make it look right.
This is how it looks now in my computer's chrome, after updating it. Could you check it again? Thanks!! 
![screenshot from 2018-04-30 18-15-13](https://user-images.githubusercontent.com/4089035/39454998-d8804d3e-4ca3-11e8-8444-9c3b1564f15f.png)